Makefile

```diff
@@ -5,6 +5,8 @@ COMPOSER=$(EXEC) composer
 CONSOLE=bin/console
 PHPCSFIXER?=$(EXEC) php -d memory_limit=1024m vendor/bin/php-cs-fixer
 BEHAT_ARGS?=-vvv
+PHPUNIT=$(EXEC) vendor/bin/phpunit
+BEHAT=$(EXEC) vendor/bin/behat
```

Comment:
Can you add a default args for the PHPUNIT_ARGS variable ?
example : `PHPUNIT_ARGS?=-v`